# Italian translation for sbackup
# Copyright (c) 2006 Rosetta Contributors and Canonical Ltd 2006
# This file is distributed under the same license as the sbackup package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2006.
#
msgid ""
msgstr ""
"Project-Id-Version: sbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2010-08-15 16:39+0200\n"
"PO-Revision-Date: 2010-05-30 16:11+0000\n"
"Last-Translator: Claudio Arseni <claudio.arseni@gmail.com>\n"
"Language-Team: Italian <it@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2010-05-31 03:44+0000\n"
"X-Generator: Launchpad (build Unknown)\n"

#: src/sbackup/backupproc.py:168 src/sbackup/pkginfo.py:63
#, fuzzy, python-format
msgid "SBackup Daemon <%(login)s@%(hostname)s>"
msgstr "Demone sbackup <%(login)s@%(hostname)s>"

#: src/sbackup/backupproc.py:172
#, fuzzy, python-format
msgid "[SBackup] [%(profile)s] Report of %(date)s"
msgstr "[sbackup] [%(profile)s] Rapporto di %(date)s"

#: src/sbackup/backupproc.py:177 src/sbackup/core/profile_handler.py:279
msgid "No log file specified."
msgstr ""

#: src/sbackup/backupproc.py:182
msgid "Unable to find log file."
msgstr ""

#: src/sbackup/backupproc.py:248
msgid ""
"Critical Error: No configuration file for the default profile was found!\n"
"\n"
"Now continue processing remaining profiles."
msgstr ""
"Errore critico: non è stato trovato alcun file di configurazione per il "
"profilo predefinito.\n"
"\n"
"Continuare elaborando i profili rimanenti."

#: src/sbackup/backupproc.py:297
msgid "Backup was canceled by user."
msgstr ""

#: src/sbackup/backupproc.py:302
msgid "An error occurred during the cancellation of backup:"
msgstr ""

#: src/sbackup/backupproc.py:308
msgid "An error occurred during the backup:"
msgstr ""

#: src/sbackup/backupproc.py:325
msgid "Error when sending email:"
msgstr ""

#: src/sbackup/backupproc.py:356
msgid "The following error(s) occurred before:"
msgstr "Si sono verificati i seguenti errori:"

#: src/sbackup/backupproc.py:535
#, python-format
msgid ""
"Backup is not being started.\n"
"%s"
msgstr ""

#: src/sbackup/pkginfo.py:43
msgid ""
"This is a user friendly backup solution for common desktop needs. If you "
"want to help by submitting bugs, by translating or anything else please "
"visit our website."
msgstr ""

#: src/sbackup/pkginfo.py:55
msgid "translator-credits"
msgstr ""
"Ubuntu Italian Translators, https://launchpad.net/~ubuntu-l10n-it, 2006-"
"2007\n"
"\n"
"Launchpad Contributions:\n"
"  Claudio Arseni https://launchpad.net/~claudio.arseni\n"
"  Expo https://launchpad.net/~expo-1993\n"
"  Jean-Peer Lorenz https://launchpad.net/~peer.loz\n"
"  Lorenzo De Liso https://launchpad.net/~blackz\n"
"  Milo Casagrande https://launchpad.net/~milo\n"
"  Simone Oberti https://launchpad.net/~simone-obo\n"
"  Stefyru91 https://launchpad.net/~stefyru91\n"
"  aleritty https://launchpad.net/~aleritty\n"
"  vinz65 https://launchpad.net/~vinz65"

#: src/sbackup/ar_backend/tar.py:108 src/sbackup/ar_backend/tar.py:160
#: src/sbackup/ar_backend/tar.py:206
msgid "Invalid archive type."
msgstr ""

#: src/sbackup/ar_backend/tar.py:327
msgid ""
"For the moment split functionality is not compatible with compress option."
msgstr ""

#: src/sbackup/ar_backend/tar.py:336 src/sbackup/ar_backend/tar.py:395
#: src/sbackup/core/profile_handler.py:289
#, python-format
msgid "Unable to change permissions for file '%s'."
msgstr "Impossibile cambiare i permessi per il file «%s»."

#: src/sbackup/ar_backend/tar.py:345
#, fuzzy, python-format
msgid "Unable to remove temporary file `%(file)s`: %(error)s"
msgstr "Ottenuto un errore con «%(file)s»: %(error)s"

#: src/sbackup/ar_backend/tar.py:362
msgid "Launching TAR to make incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:388
msgid "Unable to find the SNAR file to make an incremental backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:389
msgid "Falling back to full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:431
msgid "Launching TAR to make a full backup."
msgstr ""

#: src/sbackup/ar_backend/tar.py:495
msgid "Leading '/' from member names were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:497
msgid "Leading '/' from hard link targets were removed."
msgstr ""

#: src/sbackup/ar_backend/tar.py:506
#, python-format
msgid "TAR returned a message: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:508
#, python-format
msgid ""
"TAR returned messages:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:511
msgid "TAR has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:515
#, python-format
msgid "TAR returned a warning: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:517
#, python-format
msgid ""
"TAR returned warnings:\n"
"%s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:520
msgid "TAR returned warnings but has been finished successfully."
msgstr ""

#: src/sbackup/ar_backend/tar.py:524
msgid "Unable to finish successfully. TAR terminated with errors."
msgstr ""

#: src/sbackup/ar_backend/tar.py:705
#, python-format
msgid "Unable to terminate backend process: %s"
msgstr ""

#: src/sbackup/ar_backend/tar.py:738
msgid "Included"
msgstr "Incluso"

#: src/sbackup/ar_backend/tar.py:738
msgid "Not changed"
msgstr "Invariato"

#: src/sbackup/ar_backend/tar.py:738
msgid "Directory"
msgstr "Directory"

#: src/sbackup/ar_backend/tar.py:766
msgid "Line must be a string"
msgstr "La riga deve essere una stringa"

#: src/sbackup/ar_backend/tar.py:782
msgid "Dumpdir inconsistancy : 'filename' is empty"
msgstr "Directory di dump inconsistente: «filename» è vuoto"

#: src/sbackup/ar_backend/tar.py:795
msgid "Dumpdir inconsistancy : 'control' is empty"
msgstr "Directory di dump inconsistente: «control» è vuoto"

#: src/sbackup/ar_backend/tar.py:868
#, python-format
msgid "File '%s' does not exist."
msgstr ""

#: src/sbackup/ar_backend/tar.py:998 src/sbackup/ar_backend/tar.py:1032
#: src/sbackup/ar_backend/tar.py:1129
msgid "The snarfile header is incomplete."
msgstr ""

#: src/sbackup/ar_backend/tar.py:1298
msgid "A SnapshotFile is required"
msgstr "È richiesto un file immagine"

#: src/sbackup/ar_backend/tar.py:1398
#, python-format
msgid "Directory does not exist: %s."
msgstr ""

#: src/sbackup/core/ConfigManager.py:273
#, python-format
msgid "Profile settings are being read from file '%s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:276
#, python-format
msgid ""
"Profile settings are being set to default values. Configuration file is set "
"to '%s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:422
msgid "Option 'target' is missing, aborting."
msgstr "Opzione «target» mancante, chiusura in corso."

#: src/sbackup/core/ConfigManager.py:524 src/sbackup/core/ConfigManager.py:558
#, python-format
msgid ""
"Unable to evaluate '%(parameter)s' as a dictionary (value got = '%(value)r')."
msgstr ""

#: src/sbackup/core/ConfigManager.py:550
#, python-format
msgid ""
"Couldn't evaluate '%(parameter)s' as a dictionary (value got = '%(value)r' )"
msgstr ""
"Impossibile considerare «%(parameter)s» come un dizionario (valore ottenuto = "
"«%(value)r»)"

#: src/sbackup/core/ConfigManager.py:574
msgid "You must provide a dictionary."
msgstr "È necessario fornire un dizionario."

#: src/sbackup/core/ConfigManager.py:691
#, python-format
msgid "Log output for [%(profile)s] is directed to file '%(file)s'."
msgstr ""

#: src/sbackup/core/ConfigManager.py:698
#, python-format
msgid "Log output for [%s] is not directed into a file."
msgstr ""

#: src/sbackup/core/ConfigManager.py:717
#, python-format
msgid "Unable to rename log file '%(src)s'->'%(dst)s': %(err)s"
msgstr ""

#: src/sbackup/core/ConfigManager.py:737
#, python-format
msgid "The config file '%s' couldn't be read!"
msgstr "Il file di configurazione '%s' non può essere letto!"

#: src/sbackup/core/ConfigManager.py:749
#, python-format
msgid "section [%(section)s] in '%(configfile)s' should not exist, aborting"
msgstr ""
"sezione [%(section)s] in «%(configfile)s» inesistente, chiusura in corso"

#: src/sbackup/core/ConfigManager.py:815
msgid "No schedule defined in configuration file. Probing from filesystem."
msgstr ""

#: src/sbackup/core/ConfigManager.py:909 src/sbackup/core/ConfigManager.py:921
msgid "The config file is not set yet into this ConfigManager"
msgstr ""
"Il file di configurazione non è ancora impostato in questo ConfigManager"

#: src/sbackup/core/ConfigManager.py:1081
msgid "No receiver set."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1083
msgid "No SMTP server set."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1086
msgid "Username set but no password specified."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1089
msgid "Password set but no username specified."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1092
msgid ""
"A certificate and key file is given while SSL option (smtptls=1) is not "
"set.\n"
"Select SSL in order to use Certificate and Key."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1096
msgid ""
"When specifying a SSL certificate or key file, a key file resp. certificate "
"is mandatory."
msgstr ""

#: src/sbackup/core/ConfigManager.py:1287
msgid "Default Profile"
msgstr "Profilo predefinito"

#: src/sbackup/core/ConfigManager.py:1288
msgid "Unknown Profile"
msgstr "Profilo sconosciuto"

#: src/sbackup/core/ConfigManager.py:1338
msgid "Unlimited"
msgstr "Illimitato"

#: src/sbackup/core/ConfigManager.py:1339
msgid "100 MiB"
msgstr "100 MiB"

#: src/sbackup/core/ConfigManager.py:1340
msgid "250 MiB"
msgstr "250 MiB"

#: src/sbackup/core/ConfigManager.py:1341
msgid "650 MiB"
msgstr "650 MiB"

#: src/sbackup/core/ConfigManager.py:1342
msgid "2 GiB (FAT16)"
msgstr "2 GiB (FAT16)"

#: src/sbackup/core/ConfigManager.py:1343
msgid "4 GiB (FAT32)"
msgstr "4 GiB (FAT32)"

#: src/sbackup/core/ConfigManager.py:1344 src/sbackup/ui/configgui.py:275
#: src/sbackup/ui/configgui.py:1386
msgid "Custom"
msgstr "Personalizzato"

#: src/sbackup/core/ConfigManager.py:1346
msgid "MP3 Music"
msgstr "Musica MP3"

#: src/sbackup/core/ConfigManager.py:1347
msgid "AVI Video"
msgstr "Video AVI"

#: src/sbackup/core/ConfigManager.py:1348
#: src/sbackup/core/ConfigManager.py:1349
msgid "MPEG Video"
msgstr "Video MPEG"

#: src/sbackup/core/ConfigManager.py:1350
msgid "Matrjoshka Video"
msgstr "Video Matrjoshka"

#: src/sbackup/core/ConfigManager.py:1351
msgid "OGG Multimedia container"
msgstr "Contenitore multimediale OGG"

#: src/sbackup/core/ConfigManager.py:1352
msgid "CD Images"
msgstr "Immagini CD"

#: src/sbackup/core/RestoreManager.py:79
#, python-format
msgid "File '%s' not found in the backup snapshot files list"
msgstr ""
"Il file «%s» non è stato trovato nell'elenco dei file dell'immagine di backup"

#: src/sbackup/core/RestoreManager.py:81
#, fuzzy, python-format
msgid ""
"File '%(filename)s' not found in snapshot's [%(snapshotname)s] files list, "
"Skipped."
msgstr ""
"Il file «%(filename)s» non è stato trovato nell'elenco dei file dell'immagine "
"di backup [%(snapshotname)s]."

#: src/sbackup/core/SnapshotManager.py:125
#, python-format
msgid "Snapshot '%s' not found "
msgstr "Immagine «%s» non trovata "

#: src/sbackup/core/SnapshotManager.py:216
#, python-format
msgid "Invalid snapshot `%(name)s` found: Name of snapshot not valid."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:219
#, fuzzy, python-format
msgid "Invalid snapshot `%(name)s` found: %(error_cause)s."
msgstr "Ottenuta un'immagine non valida «%(name)s», rimozione: %(error_cause)s "

#: src/sbackup/core/SnapshotManager.py:309
#, python-format
msgid "No need to rebase a full snapshot '%s'"
msgstr "Non è necessario creare la nuova base di un'immagine completa «%s»"

#: src/sbackup/core/SnapshotManager.py:311
#, python-format
msgid ""
"'%(snapshotToRebase)s'  doesn't have 'base' file , it might have been broken "
msgstr ""
"«%(snapshotToRebase)s» non ha un file «base», potrebbe essere danneggiato "

#: src/sbackup/core/SnapshotManager.py:316
#, python-format
msgid ""
"Cannot rebase a snapshot on an earlier one : '%(snapshotToRebase)s' <= '%"
"(NewBaseSnapshot)s' "
msgstr ""
"Impossibile creare la nuova base su un'immagine precedente: «%"
"(snapshotToRebase)s» <= «%(NewBaseSnapshot)s» "

#: src/sbackup/core/SnapshotManager.py:323
#, python-format
msgid "Rebasing '%(current)s' on '%(base)s'"
msgstr "Creare la nuova base «%(current)s» su «%(base)s»"

#: src/sbackup/core/SnapshotManager.py:560
#, python-format
msgid "An error occurred when pulling snapshot '%s'."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:599
msgid "The snarfile header is incomplete !"
msgstr "L'intestazione del file snar è incompleta."

#: src/sbackup/core/SnapshotManager.py:751
#, python-format
msgid "Base of snapshot '%s' is not set."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:788
#, python-format
msgid "Snapshot '%s' is already Full, nothing to do (not changing it to full)."
msgstr ""

#: src/sbackup/core/SnapshotManager.py:828
#, python-format
msgid "Cancelling pull of snapshot '%s'"
msgstr "Annullamento del prelevamento dell'immagine «%s»"

#: src/sbackup/core/SnapshotManager.py:998
msgid ""
"Logarithmic purge is currently disabled. Please specify a cutoff purge "
"interval in your configuration instead."
msgstr ""

#: src/sbackup/core/UpgradeManager.py:172
#, python-format
msgid "Upgrading from v1.0 to v1.2: %s"
msgstr "Aggiornamento dalla versione 1.0 alla 1.2: %s"

#: src/sbackup/core/UpgradeManager.py:189
#, python-format
msgid "Upgrading to v1.3: %s"
msgstr "Aggiornamento alla versione 1.3: %s"

#: src/sbackup/core/UpgradeManager.py:214
#, python-format
msgid "Upgrading to v1.4: %s"
msgstr "Aggiornamento alla versione 1.4: %s"

#: src/sbackup/core/UpgradeManager.py:228
#, python-format
msgid "Upgrading to v1.5: %s"
msgstr "Aggiornamento alla versione 1.5: %s"

#: src/sbackup/core/UpgradeManager.py:282
#, python-format
msgid "The SNAR file already exist for snapshot '%s'. It is not overwritten."
msgstr ""

#: src/sbackup/core/profile_handler.py:90
#, fuzzy
msgid "Preparation of backup process"
msgstr "Avvio della sessione di backup"

#: src/sbackup/core/profile_handler.py:134
msgid ""
"There are snapshots stored in outdated snapshot formats. Please upgrade them "
"using 'Simple Backup-Restoration' if you want to use them."
msgstr ""

#: src/sbackup/core/profile_handler.py:143
#, python-format
msgid "Snapshot '%(name)s' is being made."
msgstr ""

#: src/sbackup/core/profile_handler.py:151
#, python-format
msgid "Setting Base to '%(value)s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:160
msgid "Setting packages File."
msgstr ""

#: src/sbackup/core/profile_handler.py:166
msgid "Problem when setting the packages list: "
msgstr ""

#: src/sbackup/core/profile_handler.py:170
msgid "Setting Excludes File."
msgstr ""

#: src/sbackup/core/profile_handler.py:179
#, python-format
msgid "Setting compression format to `%s`"
msgstr ""

#: src/sbackup/core/profile_handler.py:186
#, python-format
msgid "Setting size of archive chunks to %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:193
msgid "Option 'Follow symbolic links' is enabled."
msgstr ""

#: src/sbackup/core/profile_handler.py:195
msgid "Option 'Follow symbolic links' is disabled."
msgstr ""

#: src/sbackup/core/profile_handler.py:204
#, fuzzy
msgid "Snapshot is being committed"
msgstr "È richiesto un file immagine"

#: src/sbackup/core/profile_handler.py:218
#, python-format
msgid "Error while purging old snapshots: %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:220
msgid "Backup process finished."
msgstr ""

#: src/sbackup/core/profile_handler.py:227
msgid "Inspect file system and collect file infos"
msgstr ""

#: src/sbackup/core/profile_handler.py:238
msgid "Summary of backup"
msgstr ""

#: src/sbackup/core/profile_handler.py:239
#, python-format
msgid "Number of directories: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:240
#, python-format
msgid "Total number of files: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:241
#, python-format
msgid "Number of symlinks: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:242
#, python-format
msgid "Number of files included in snapshot: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:243
#, python-format
msgid "Number of new files (also included): %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:244
#, python-format
msgid "Number of files skipped in incremental snapshot: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:245
#, python-format
msgid "Number of items forced to be excluded: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:246
#, python-format
msgid "Number of items to be excluded by config: %s."
msgstr ""

#: src/sbackup/core/profile_handler.py:247
#, python-format
msgid "Maximum free size required is '%s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:253
#, python-format
msgid "Available disk size is '%s'."
msgstr ""

#: src/sbackup/core/profile_handler.py:255
#, python-format
msgid ""
"Not enough free space in the target directory for the planned backup (%"
"(freespace)s <= %(neededspace)s)."
msgstr ""

#: src/sbackup/core/profile_handler.py:274
msgid "Unable to copy log. File access is not initialized."
msgstr ""

#: src/sbackup/core/profile_handler.py:292
#, fuzzy, python-format
msgid "Unable to copy log file: %s"
msgstr "Nessuna immagine per copiare il file di registro"

#: src/sbackup/core/profile_handler.py:294
msgid "Unable to find logfile to copy into snapshot."
msgstr "Impossibile trovare il logfile da copiare in snapshot."

#: src/sbackup/core/profile_handler.py:296
msgid "No snapshot to copy logfile."
msgstr "Nessuna immagine per copiare il file di registro"

#: src/sbackup/core/profile_handler.py:303
msgid "Processing of profile was canceled on user request\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:320
msgid "Processing of profile successfully finished (no errors)\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:326
#, python-format
msgid "Processing of profile failed with error: %s\n"
msgstr ""

#: src/sbackup/core/profile_handler.py:337
#, fuzzy, python-format
msgid "Backup destination: %s"
msgstr "Ripristino di sbackup"

#: src/sbackup/core/profile_handler.py:341
#, fuzzy
msgid "Unable to find destination directory."
msgstr "Impossibile aprire la directory di destinazione.\n"

#: src/sbackup/core/profile_handler.py:360
msgid "Unable to find destination directory even after retry."
msgstr ""

#: src/sbackup/core/profile_handler.py:361
#: src/sbackup/core/profile_handler.py:368
#, python-format
msgid "Target directory '%(target)s' does not exist."
msgstr "La cartella bersaglio '%(target)s' non esiste."

#: src/sbackup/core/profile_handler.py:374
#, python-format
msgid "Unable to access destination: %s"
msgstr ""

#: src/sbackup/core/snapshot.py:129
msgid "Snapshot is inconsistent: __name is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:140
#, python-format
msgid "Name of snapshot '%s' doesn't match requirement"
msgstr "Il nome dell'immagine «%s» non corrisponde ai requisiti"

#: src/sbackup/core/snapshot.py:234
msgid "Snapshot is inconsistent: __snapshotpath is not set "
msgstr ""

#: src/sbackup/core/snapshot.py:321
#, python-format
msgid ""
"The snapshot compression format is supposed to be '%s' but the corresponding "
"well named file wasn't found"
msgstr ""
"Il formato di compressione delle immagini dovrebbe essere «%s», ma il file "
"corrispondente non è stato trovato"

#: src/sbackup/core/snapshot.py:345
#, python-format
msgid ""
"%(file)s doesn't contain valid value. Ignoring incomplete or non - backup "
"directory. "
msgstr ""

#: src/sbackup/core/snapshot.py:482
#, python-format
msgid "Name of Snapshot not valid : %s"
msgstr "Nome dell'immagine non valido: %s"

#: src/sbackup/core/snapshot.py:500
#, python-format
msgid "Name of base not valid : %s"
msgstr "Nome della base non valido: %s"

#: src/sbackup/core/snapshot.py:546
#, fuzzy, python-format
msgid "Invalid name of snapshot: %s"
msgstr "Annullamento del prelevamento dell'immagine «%s»"

#: src/sbackup/core/snapshot.py:550
msgid "Unable to read mandatory file `ver`"
msgstr ""

#: src/sbackup/core/snapshot.py:595
msgid "Base name must be set for incremental backup."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:106
msgid "Initializing FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_fam.py:129
msgid "Terminating FUSE File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:44
#: src/sbackup/fs_backend/_gio_utils.py:106
msgid "SFTP"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:45
#: src/sbackup/fs_backend/_gio_utils.py:107
msgid "SSH"
msgstr ""

#: src/sbackup/fs_backend/_fuse_utils.py:46
#: src/sbackup/fs_backend/_gio_utils.py:108
msgid "FTP"
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:96
msgid "Initializing GIO File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_gio_fam.py:110
msgid "Terminating GIO File Access Manager."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:109
msgid "NFS"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:187
msgid "Unable to get mount: path is not mounted"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:189
msgid "Unable to get mount: path not found when mounting (is probably local)"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:191
#: src/sbackup/fs_backend/_gio_utils.py:194
#, fuzzy, python-format
msgid "Unable to get mount: %s"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/fs_backend/_gio_utils.py:241
msgid "Unable to un-set mount flag: path is not stored"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:355
msgid "Path is already mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:378
msgid "Path is not mounted."
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:809
msgid "test specified path for existence using GIO"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:822
#, fuzzy
msgid "Create testdir"
msgstr "Creazione della directory di destinazione «%s»"

#: src/sbackup/fs_backend/_gio_utils.py:827
msgid "Test testfile for existence"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:834
#, fuzzy
msgid "Create file"
msgstr "Profilo predefinito"

#: src/sbackup/fs_backend/_gio_utils.py:841
msgid "Re-read test file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:853
msgid "Remove file"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:855
#, fuzzy
msgid "Remove dir"
msgstr "rimuovi"

#: src/sbackup/fuse_plugins/__init__.py:160
#, fuzzy, python-format
msgid "Unable to import plugin `%(plugin_name)s`: %(error_cause)s "
msgstr ""
"Impossibile importare il plugin %(plugin_name)s. Causa: %(error_cause)s "

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:117
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of ftp "
"sites: %s"
msgstr ""
"Impossibile trovare l'applicazione esterna «curlftpfs» necessaria per "
"l'utilizzo di siti ftp: %s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:127
#: src/sbackup/fuse_plugins/sftpFuseFAM.py:124
#, python-format
msgid "Couldn't mount '%(server)s' into '%(mountpoint)s' : %(error)s"
msgstr "Impossibile montare «%(server)s» in «%(mountpoint)s»: %(error)s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:141
msgid "FTP schema is: ftp://user:pass@server/anything"
msgstr ""

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:188
#, python-format
msgid ""
"Error matching the schema 'ftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Errore di corrispondenza tra «ftp://utente:password@server/cartella» e «%"
"s» (la «/» dopo server è obbligatoria)"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:114
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of sftp "
"sites: %s"
msgstr ""

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:138
msgid "SFTP schema is like : sftp://user:pass@server/anything"
msgstr "Un indirizzo SFTP è simile a: sftp://utente:password@server/cartella"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:185
#, python-format
msgid ""
"Error matching the schema 'sftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Errore di corrispondenza tra «sftp://utente:password@server/cartella» e «%"
"s» (la «/» dopo server è obbligatoria)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:81
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com/home/' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Errore di corrispondenza tra «ssh://utente:password@esempio.it/home/» e «%"
"s» (la «/» dopo server è obbligatoria)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:137
#, python-format
msgid ""
"The sshfs command '%(command)s' didn't perform normally. Output => %"
"(erroroutput)s "
msgstr ""
"Il comando sshfd '%(command)s' non viene eseguito normalmente. Output => %"
"(erroroutput)s "

#: src/sbackup/fuse_plugins/sshFuseFAM.py:142
msgid "SSH schema is like : ssh://user:pass@example.com:33/home/user/backup/"
msgstr ""
"Lo schema SSH è simile a  : ssh://user:pass@example.com:33/home/user/backup"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:158
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com:21/home/' with '%"
"s' (The '/' after server is mandatory)"
msgstr ""
"Errore nella corrispondenza dello schema 'ssh://user:pass@example.com:21/"
"home' con '%s' (Lo '/' dopo il server è obbligatorio)"

#: src/sbackup/ui/GladeWindow.py:201
#, fuzzy
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target."
msgstr ""
"Assicurarsi che la directory mancante esista (es. montando un disco esterno) "
"o cambiare la destinazione con una esistente."

#: src/sbackup/ui/GladeWindow.py:202
#, python-format
msgid ""
"While accessing the backup destination following error occurred:\n"
"\n"
"%s."
msgstr ""

#: src/sbackup/ui/GladeWindow.py:203 src/sbackup/ui/restoregui.py:783
#: src/sbackup/ui/restoregui.py:837
msgid "Simple Backup error"
msgstr ""

#: src/sbackup/ui/GladeWindow.py:204
#, fuzzy
msgid "Unable to access backup destination"
msgstr "Impossibile aprire l'oggetto di backup"

#: src/sbackup/ui/configgui.py:121
msgid "Scheduled backups are available for Administrator users only."
msgstr ""

#: src/sbackup/ui/configgui.py:131 src/sbackup/ui/configgui.py:140
msgid "Name"
msgstr "Nome"

#: src/sbackup/ui/configgui.py:148
msgid "File Type"
msgstr "Tipo di file"

#: src/sbackup/ui/configgui.py:183
msgid "Enable"
msgstr "Abilita"

#: src/sbackup/ui/configgui.py:184
msgid "Profile Name"
msgstr "Nome del profilo"

#: src/sbackup/ui/configgui.py:281
#, python-format
msgid ""
"Invalid or empty regular expression ('%s') found in configuration. Removed."
msgstr ""

#: src/sbackup/ui/configgui.py:288
#, python-format
msgid ""
"Invalid or empty regular expressions found\n"
"in configuration file:\n"
"'%s'\n"
"\n"
"These expressions are not used and were\n"
"removed from the configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:535
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target to an existing one."
msgstr ""
"Assicurarsi che la directory mancante esista (es. montando un disco esterno) "
"o cambiare la destinazione con una esistente."

#: src/sbackup/ui/configgui.py:536
msgid ""
"Backup target does not exist.\n"
"\n"
"Attention: The target will be set to the default value. Check this on the "
"destination settings page before saving the configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:538
msgid "Unable to open backup target"
msgstr "Impossibile aprire l'oggetto di backup"

#: src/sbackup/ui/configgui.py:609
#, python-format
msgid "Current profile: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:700
msgid "Already included item !"
msgstr "Oggetto già incluso."

#: src/sbackup/ui/configgui.py:715
msgid "Already excluded item !"
msgstr "Oggetto già escluso."

#: src/sbackup/ui/configgui.py:727 src/sbackup/ui/configgui.py:1426
msgid "Empty expression. Please enter a valid regular expression."
msgstr "Espressione vuota. Inserire un'espressione regolare valida."

#: src/sbackup/ui/configgui.py:742 src/sbackup/ui/configgui.py:1440
msgid "Provided regular expression is not valid."
msgstr "L'espressione regolare fornita non è valida."

#: src/sbackup/ui/configgui.py:750
msgid "Empty filename or path. Please enter a valid filename or path."
msgstr ""
"Nome file o per percorso vuoto. Inserire un nome file o un percorso valido."

#: src/sbackup/ui/configgui.py:759
msgid ""
"It seems the path you entered does not exists. Do you want to add this "
"incorrect path?"
msgstr ""

#: src/sbackup/ui/configgui.py:788
msgid "Save configuration as..."
msgstr ""

#: src/sbackup/ui/configgui.py:855
msgid ""
"A backup process is now executed in the background.\n"
"\n"
"You can monitor the progress of the backup by means of the status indicator "
"displayed in the notification area."
msgstr ""

#: src/sbackup/ui/configgui.py:856
msgid "Backup process started"
msgstr ""

#: src/sbackup/ui/configgui.py:861 src/sbackup/ui/configgui.py:1629
#: src/sbackup/ui/configgui.py:1999
#, python-format
msgid "Unexpected dialog response: %s"
msgstr ""

#: src/sbackup/ui/configgui.py:935
msgid "Include file..."
msgstr ""

#: src/sbackup/ui/configgui.py:942 src/sbackup/ui/configgui.py:997
msgid "All files"
msgstr "Tutti i file"

#: src/sbackup/ui/configgui.py:962
msgid "Include folder..."
msgstr ""

#: src/sbackup/ui/configgui.py:990
msgid "Exclude file..."
msgstr ""

#: src/sbackup/ui/configgui.py:1013
msgid "Exclude folder..."
msgstr ""

#: src/sbackup/ui/configgui.py:1278
msgid "Configuration has changed"
msgstr ""

#: src/sbackup/ui/configgui.py:1279
msgid ""
"There are unsaved modifications. Please save the configuration or revert "
"these changes before testing the mail settings."
msgstr ""

#: src/sbackup/ui/configgui.py:1285 src/sbackup/ui/configgui.py:1291
msgid "Test mail settings"
msgstr "Provare le impostazioni delle email"

#: src/sbackup/ui/configgui.py:1286
msgid "The test failed with following output:"
msgstr ""

#: src/sbackup/ui/configgui.py:1292
msgid "The test was successful."
msgstr ""

#: src/sbackup/ui/configgui.py:1375 src/sbackup/ui/configgui.py:1418
#, python-format
msgid ""
"The given expression contains unsupported characters ('%s'). Currently it is "
"not possible to use these characters in exclude expressions."
msgstr ""

#: src/sbackup/ui/configgui.py:1378 src/sbackup/ui/configgui.py:1421
msgid "Unsupported character"
msgstr ""

#: src/sbackup/ui/configgui.py:1589
#, fuzzy
msgid "Connect to remote destination"
msgstr "Selezionare la cartella di destinazione"

#: src/sbackup/ui/configgui.py:1637
msgid "Mount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1648
#, fuzzy
msgid "Unable to mount host"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/ui/configgui.py:1657
msgid "Umount was sucessful (no errors)"
msgstr ""

#: src/sbackup/ui/configgui.py:1667
#, fuzzy
msgid "Unable to unmount host"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/ui/configgui.py:1697
msgid "Perfom tests on remote host"
msgstr ""

#: src/sbackup/ui/configgui.py:1710
msgid "Unable to access remote destination"
msgstr ""

#: src/sbackup/ui/configgui.py:1714
msgid "All tests passed"
msgstr ""

#: src/sbackup/ui/configgui.py:1871
msgid ""
"The given name of the new profile is empty. Please enter a valid profile "
"name."
msgstr ""

#: src/sbackup/ui/configgui.py:1872 src/sbackup/ui/configgui.py:1879
msgid "Profile name not valid"
msgstr ""

#: src/sbackup/ui/configgui.py:1878
msgid ""
"The given name of the new profile already exists. Please enter another name."
msgstr ""

#: src/sbackup/ui/configgui.py:1880
msgid "Renaming of profiles is not supported."
msgstr ""

#: src/sbackup/ui/configgui.py:1910
#, python-format
msgid ""
"<b>Delete configuration profile?</b>\n"
"\n"
"You are trying to remove a configuration profile. You will not be able to "
"restore it. If you are not sure, use the 'enable|disable' functionality "
"instead.\n"
"\n"
"Do you really want to delete the profile '%(name)s'?"
msgstr ""

#: src/sbackup/ui/configgui.py:1983
#, python-format
msgid ""
"<big><b>Set default values for current profile?</b></big>\n"
"This will restore the default values for the profile currently edited: '%"
"s'.\n"
"\n"
"These predefined settings are recommended for most users. Check whether they "
"are appropriate for your use before saving the changed configuration."
msgstr ""

#: src/sbackup/ui/configgui.py:1993
msgid "Default settings are being applied."
msgstr ""

#: src/sbackup/ui/configgui.py:2018
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be removed. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2032
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be disabled. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""

#: src/sbackup/ui/configgui.py:2045
#, fuzzy
msgid "Please select a profile."
msgstr "Selezionare un profilo."

#: src/sbackup/ui/indicator_core.py:116
#, fuzzy
msgid "there was a problem initializing the pynotify module"
msgstr "si è verificato un errore inizializzando "

#: src/sbackup/ui/indicator_core.py:145 src/sbackup/ui/indicator_core.py:192
#: src/sbackup/ui/indicator_core.py:220 src/sbackup/ui/indicator_core.py:242
#, python-format
msgid "Connection to notification-daemon failed: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:305
msgid "Simple Backup"
msgstr ""

#: src/sbackup/ui/indicator_core.py:311
msgid "Show message windows"
msgstr ""

#: src/sbackup/ui/indicator_core.py:353 src/sbackup/ui/indicator_core.py:807
#, fuzzy
msgid "Cancel Backup"
msgstr "Eliminazione vecchi backup"

#: src/sbackup/ui/indicator_core.py:415
msgid "Connection to D-Bus service lost."
msgstr ""

#: src/sbackup/ui/indicator_core.py:440
msgid "re-connection was successful"
msgstr ""

#: src/sbackup/ui/indicator_core.py:479
msgid ""
"No backup in progress and connection to D-Bus service lost. Simple Backup "
"Indicator is being terminated."
msgstr ""

#: src/sbackup/ui/indicator_core.py:508
#, fuzzy
msgid "Starting backup session"
msgstr "Avvio della sessione di backup"

#: src/sbackup/ui/indicator_core.py:522
#, fuzzy
msgid "Ending Backup Session"
msgstr "Finendo la Sessione di Backup."

#: src/sbackup/ui/indicator_core.py:535
msgid ""
"There are snapshots with old snapshot format. Please upgrade these using the "
"Restoration tool if you want to use them."
msgstr ""

#: src/sbackup/ui/indicator_core.py:537
#, python-format
msgid "Unknown D-Bus event `%s` received."
msgstr ""

#: src/sbackup/ui/indicator_core.py:578
msgid "Try again"
msgstr ""

#: src/sbackup/ui/indicator_core.py:642
msgid ""
"Attempt of starting another instance of Simple Backup while this one is "
"already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:652
#, python-format
msgid "Unable to present window: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:668
#, fuzzy, python-format
msgid "Do you really want to cancel the backup process of profile '%s'?"
msgstr "Creare la nuova base «%(current)s» su «%(base)s»?"

#: src/sbackup/ui/indicator_core.py:672
#, fuzzy
msgid "Cancel running backup process?"
msgstr "Avvio della sessione di backup"

#: src/sbackup/ui/indicator_core.py:689
#, fuzzy
msgid "Unable to cancel backup process."
msgstr "Impossibile aprire l'oggetto di backup"

#: src/sbackup/ui/indicator_core.py:751
#, fuzzy, python-format
msgid "Profile: %s"
msgstr "Profili"

#: src/sbackup/ui/indicator_core.py:752
#, python-format
msgid "Size of backup: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:753
#, python-format
msgid "%.1f%% processed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:754
#, python-format
msgid "Remaining time: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:800
#, python-format
msgid "PID of backup process being canceled: %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:805
#, python-format
msgid "Unable to send signal to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:806
#, python-format
msgid "Sending signal using gksu to process %s"
msgstr ""

#: src/sbackup/ui/indicator_core.py:810
#, python-format
msgid "Sending signal using gksu to process %s failed"
msgstr ""

#: src/sbackup/ui/indicator_core.py:845
#, python-format
msgid ""
"<b>Unable to find specified target directory</b>\n"
"\n"
"The specified target directory '%s' was not found.\n"
"\n"
msgstr ""

#: src/sbackup/ui/indicator_core.py:846
#, python-format
msgid ""
"You can try to use the specified target again or cancel the profile "
"execution. The profile execution is canceled automatically in %s seconds."
msgstr ""

#: src/sbackup/ui/indicator_core.py:872 src/sbackup/ui/indicator_core.py:914
#: src/sbackup/ui/indicator_core.py:932 src/sbackup/ui/indicator_core.py:940
#: src/sbackup/ui/indicator_core.py:957
msgid "unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:893
msgid "less than 1 minute"
msgstr ""

#: src/sbackup/ui/indicator_core.py:895
msgid "about 2 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:897
msgid "about 3 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:899
msgid "about 4 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:901
msgid "about 5 minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:903
#, python-format
msgid "about %.0f minutes"
msgstr ""

#: src/sbackup/ui/indicator_core.py:913
msgid "In progress"
msgstr ""

#: src/sbackup/ui/indicator_core.py:923 src/sbackup/ui/indicator_core.py:949
msgid "finished"
msgstr ""

#: src/sbackup/ui/indicator_core.py:931
msgid "Progress: unknown"
msgstr ""

#: src/sbackup/ui/indicator_core.py:938
#, fuzzy
msgid "Backup is being prepared"
msgstr "BackupManager creato "

#: src/sbackup/ui/indicator_core.py:939
msgid "Collecting file info"
msgstr ""

#: src/sbackup/ui/indicator_core.py:948
msgid "Progress: canceled"
msgstr ""

#: src/sbackup/ui/indicator_core.py:968
msgid "An error occurred"
msgstr ""

#: src/sbackup/ui/indicator_core.py:1001
msgid "Another `Simple Backup Indicator` is already running."
msgstr ""

#: src/sbackup/ui/indicator_core.py:1025
#, python-format
msgid "Initialization of GTK+ failed: %s"
msgstr ""

#: src/sbackup/ui/misc.py:293
#, python-format
msgid "%s   (Administrator mode)"
msgstr ""

#: src/sbackup/ui/misc.py:323
#, python-format
msgid ""
"An uncaught error occurred. Close this message window and restart the "
"application.\n"
"\n"
"Please report this error on https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"
msgstr ""

#: src/sbackup/ui/misc.py:324
msgid "Sorry, this should not have happened"
msgstr ""

#: src/sbackup/ui/restoregui.py:77
msgid "Restore..."
msgstr "Ripristina..."

#: src/sbackup/ui/restoregui.py:78 data/ui/sbackup-restore.glade:845
msgid "Restore as..."
msgstr "Ripristina come..."

#: src/sbackup/ui/restoregui.py:79
msgid "Revert..."
msgstr "Reimposta..."

#: src/sbackup/ui/restoregui.py:80 data/ui/sbackup-restore.glade:877
msgid "Revert as..."
msgstr "Reimposta come..."

#: src/sbackup/ui/restoregui.py:127 data/ui/sbackup-restore.glade:111
#: data/ui/sbackup-restore.glade:591
msgid "n.a."
msgstr ""

#: src/sbackup/ui/restoregui.py:145 src/sbackup/ui/restoregui.py:162
msgid "Snapshots"
msgstr "Immagini"

#: src/sbackup/ui/restoregui.py:146
msgid "Version"
msgstr "Versione"

#: src/sbackup/ui/restoregui.py:153 data/ui/sbackup-config.glade:4063
msgid "Path"
msgstr "Percorso"

#: src/sbackup/ui/restoregui.py:158
msgid "State"
msgstr "Stato"

#: src/sbackup/ui/restoregui.py:177
#, fuzzy
msgid "Connect to destination..."
msgstr "Selezionare la cartella di destinazione"

#: src/sbackup/ui/restoregui.py:201
msgid "Specified path does not exist"
msgstr ""

#: src/sbackup/ui/restoregui.py:246
msgid "Ready"
msgstr "Pronto"

#: src/sbackup/ui/restoregui.py:360
msgid "No destination specified"
msgstr ""

#: src/sbackup/ui/restoregui.py:377
msgid "Choose a source folder"
msgstr "Scegliere una cartella"

#: src/sbackup/ui/restoregui.py:394
#, fuzzy
msgid "Reading snapshots..."
msgstr "Lettura dell'immagine di backup..."

#: src/sbackup/ui/restoregui.py:445 src/sbackup/ui/restoregui.py:483
msgid "Loading ..."
msgstr "Caricamento..."

#: src/sbackup/ui/restoregui.py:532
msgid "Simple Backup restore error"
msgstr ""

#: src/sbackup/ui/restoregui.py:533
msgid "An error occurred while reading snapshot"
msgstr ""

#: src/sbackup/ui/restoregui.py:538
msgid "This snapshot seems empty."
msgstr "Questa immagine sembra vuota."

#: src/sbackup/ui/restoregui.py:555
msgid "Reading backup snapshot..."
msgstr "Lettura dell'immagine di backup..."

#: src/sbackup/ui/restoregui.py:578
msgid "No backups found for this day."
msgstr ""

#: src/sbackup/ui/restoregui.py:593
#, fuzzy, python-format
msgid ""
"Snapshot version is not supported (Only %(supportedversion)s is supported). "
"Version '%(currentversion)s' found. You should upgrade it. "
msgstr ""
"Questa versione dell'immagine non è supportata (solo %(supportedversion)s è "
"supportata). Versione trovata «%(currentversion)s». Si consiglia di "
"effettuare l'aggiornamento. "

#: src/sbackup/ui/restoregui.py:620
msgid "Select restore location"
msgstr "Seleziona posizione di ripristino"

#: src/sbackup/ui/restoregui.py:638
#, python-format
msgid "Do you really want to revert '%s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:651
msgid "Select revert location"
msgstr "Selezionare la posizione di reimpostazione"

#: src/sbackup/ui/restoregui.py:660
#, python-format
msgid "Do you really want to revert '%(source)s' to '%(dir)s'?"
msgstr ""

#: src/sbackup/ui/restoregui.py:738
#, fuzzy
msgid "Reading snapshot..."
msgstr "Lettura dell'immagine di backup..."

#: src/sbackup/ui/restoregui.py:757
msgid "The version of the snapshot is greater than the supported one!"
msgstr "La versione dell'immagine è superiore a quella supportata."

#: src/sbackup/ui/restoregui.py:767
#, python-format
msgid ""
"Upgrading the selected snapshot `%s` might corrupt your backup for this "
"particular point in time. You cannot undo this operation. It is recommended "
"to create a copy of the snapshot directory before upgrading it. The upgrade "
"process takes a while - be patient.\n"
"\n"
"Are you sure that you want to upgrade the snapshot now?"
msgstr ""

#: src/sbackup/ui/restoregui.py:771
#, fuzzy
msgid "Do you really want to upgrade snapshot?"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/ui/restoregui.py:781
#, python-format
msgid ""
"While attempting to upgrade snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:784
msgid "Unable to upgrade snapshot"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/ui/restoregui.py:804
#, python-format
msgid "Do you really want to rebase '%(current)s' on '%(base)s' ?"
msgstr "Creare la nuova base «%(current)s» su «%(base)s»?"

#: src/sbackup/ui/restoregui.py:822
#, python-format
msgid ""
"Deleting the selected snapshot `%s` will remove your backup for this "
"particular point in time. You cannot undo this operation.\n"
"\n"
"Are you sure that you want to remove the snapshot permanently?"
msgstr ""

#: src/sbackup/ui/restoregui.py:826
#, fuzzy
msgid "Do you really want to remove snapshot?"
msgstr "Creare la nuova base «%(current)s» su «%(base)s»?"

#: src/sbackup/ui/restoregui.py:835
#, python-format
msgid ""
"While attempting to delete snapshot the following error occurred:\n"
"%s"
msgstr ""

#: src/sbackup/ui/restoregui.py:838
#, fuzzy
msgid "Unable to delete snapshot"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/ui/restoregui.py:844
#, python-format
msgid ""
"The selected snapshot '%s' is referenced by more recent snapshots as base "
"snapshot.\n"
"\n"
"Remove all of these child snapshots in order to delete this snapshot."
msgstr ""

#: src/sbackup/ui/restoregui.py:846
#, fuzzy
msgid "Unable to remove snapshot"
msgstr "Impossibile aggiornare l'immagine"

#: src/sbackup/ui/restoregui.py:854
msgid "Termination in progress"
msgstr ""

#: src/sbackup/ui/restoregui.py:870 src/sbackup/ui/restoregui.py:876
#: src/sbackup/ui/restoregui.py:882 src/sbackup/ui/restoregui.py:888
#: data/ui/sbackup-restore.glade:1039
#, fuzzy
msgid "Simple Backup restoration"
msgstr "Ripristino di sbackup"

#: src/sbackup/ui/restoregui.py:871 src/sbackup/ui/restoregui.py:877
msgid "<b>Restoring of selected files</b>"
msgstr "<b>Ripristino del file selezionato</b>"

#: src/sbackup/ui/restoregui.py:872
#, python-format
msgid "Restoring of <tt>'%s'</tt> is in progress."
msgstr "Ripristino di <tt>«%s»</tt> in corso."

#: src/sbackup/ui/restoregui.py:873
#, python-format
msgid "Restoring of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:874
#, fuzzy, python-format
msgid ""
"Restoring of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr "Ripristino di <tt>«%s»</tt> in corso."

#: src/sbackup/ui/restoregui.py:878
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Ripristino di <tt>«%(source)s»</tt>\n"
"in <tt>«%(dirname)s»</tt> in corso."

#: src/sbackup/ui/restoregui.py:879
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:880
#, fuzzy, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"Ripristino di <tt>«%(source)s»</tt>\n"
"in <tt>«%(dirname)s»</tt> in corso."

#: src/sbackup/ui/restoregui.py:883 src/sbackup/ui/restoregui.py:889
msgid "<b>Reverting selected files</b>"
msgstr "<b>Reimpostazione dei file selezionati</b>"

#: src/sbackup/ui/restoregui.py:884
#, python-format
msgid "Reverting of <tt>'%s'</tt> is in progress.\n"
msgstr "Reimpostazione di <tt>«%s»</tt> in corso.\n"

#: src/sbackup/ui/restoregui.py:885
#, python-format
msgid "Reverting of <tt>'%s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:886
#, fuzzy, python-format
msgid ""
"Reverting of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr "Reimpostazione di <tt>«%s»</tt> in corso.\n"

#: src/sbackup/ui/restoregui.py:890
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Reimpostazione di <tt>«%(source)s»</tt>\n"
"in <tt>«%(dirname)s»</tt> in corso."

#: src/sbackup/ui/restoregui.py:891
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""

#: src/sbackup/ui/restoregui.py:892
#, fuzzy, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"Reimpostazione di <tt>«%(source)s»</tt>\n"
"in <tt>«%(dirname)s»</tt> in corso."

#: src/sbackup/util/__init__.py:439
#, python-format
msgid "%(mb)d MiB %(kb)d KiB"
msgstr ""

#: src/sbackup/util/__init__.py:442
#, python-format
msgid "%(mb)d MiB %(kb)d KiB %(bytes)d"
msgstr ""

#: src/sbackup/util/__init__.py:447
#, python-format
msgid "%(mb)d MB %(kb)d kB"
msgstr ""

#: src/sbackup/util/__init__.py:450
#, python-format
msgid "%(mb)d MB %(kb)d kB %(bytes)d"
msgstr ""

#: src/sbackup/util/filecollect.py:335
#, python-format
msgid "File '%(file)s' does not exist."
msgstr ""

#: src/sbackup/util/filecollect.py:348
#, python-format
msgid "File '%(file)s' is not accessable with error '%(error)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:364
#, python-format
msgid "File '%(file)s' cannot be opened for read access. Operation timed out."
msgstr ""

#: src/sbackup/util/filecollect.py:368
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access with error '%(error)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:385
#, python-format
msgid "File '%(file)s' is backup's target directory."
msgstr ""

#: src/sbackup/util/filecollect.py:394
#, python-format
msgid "Symbolic link '%(path)s' -> '%(ln_target)s' is a circular symlink."
msgstr ""

#: src/sbackup/util/filecollect.py:414
#, python-format
msgid ""
"File '%(file)s' exceeds maximum file size ( %(filesize)s > %(maxsize)s)."
msgstr ""

#: src/sbackup/util/filecollect.py:424
#, python-format
msgid "File '%(file)s' matches regular expression '%(regex)s'."
msgstr ""

#: src/sbackup/util/filecollect.py:430
#, python-format
msgid "Path '%(file)s' defined in excludes list."
msgstr ""

#: src/sbackup/util/filecollect.py:501
#, python-format
msgid "Error while checking directory '%(dir)s': %(error)s."
msgstr ""

#: src/sbackup/util/filecollect.py:552
msgid "Empty regular expression found. Skipped."
msgstr ""

#: src/sbackup/util/filecollect.py:558
#, python-format
msgid "Invalid regular expression ('%s') found. Skipped."
msgstr ""

#: src/sbackup/util/filecollect.py:577
msgid "No directories to backup defined."
msgstr ""

#: src/sbackup/util/lock.py:76
msgid "Another application instance is already running."
msgstr ""

#: src/sbackup/util/lock.py:78
msgid "Invalid lock file found. Is being removed."
msgstr ""

#: src/sbackup/util/lock.py:128 src/sbackup/util/lock.py:142
#, python-format
msgid "Unable to remove lock file: %s"
msgstr ""

#: src/sbackup/util/lock.py:132 src/sbackup/util/lock.py:144
msgid "Unable to remove lock file: File not found."
msgstr ""

#: data/ui/sbackup-config.glade:11
msgid "Exclude file type"
msgstr "Escludi tipo di file"

#: data/ui/sbackup-config.glade:26
msgid "Standard file types:"
msgstr "Tipi di file standard:"

#: data/ui/sbackup-config.glade:63
msgid "Custom defined file type with "
msgstr "Tipo di file definito dall'utente con "

#: data/ui/sbackup-config.glade:93
msgid "file extension"
msgstr "estensione del file"

#: data/ui/sbackup-config.glade:163
msgid "Exclude by regex"
msgstr "Escludere con una espressione regolare"

#: data/ui/sbackup-config.glade:180
msgid ""
"Enter a regular expression.\n"
"Files and directories matching this expression will be excluded from backup."
msgstr ""
"Inserire un'espressione regolare.\n"
"I file e le directory corrispondenti a a questa espressione verranno esclusi "
"dal backup."

#: data/ui/sbackup-config.glade:257
msgid "Enter the remote target to include"
msgstr "Inserire la destinazione remota da includere"

#: data/ui/sbackup-config.glade:278
msgid "Choose a plugin to get the format"
msgstr "Scegliere un plugin per ottenere il formato"

#: data/ui/sbackup-config.glade:295
msgid "Choose your plugin :"
msgstr "Scegliere il plugin:"

#: data/ui/sbackup-config.glade:367
msgid "Check"
msgstr "Controllo"

#: data/ui/sbackup-config.glade:479
msgid ""
"<b>Save changed configuration?</b>\n"
"\n"
"The configuration has changed. If you want to use the modified "
"configuration, you have to save it.\n"
"\n"
"Do you want to save the configuration now?"
msgstr ""

#: data/ui/sbackup-config.glade:572
msgid "Configuration Profile Manager"
msgstr ""

#: data/ui/sbackup-config.glade:594
msgid ""
"You can set up multiple configuration profiles which are processed when "
"performing a backup. Each profile has its own set of settings including "
"backup destination, backup format etc."
msgstr ""

#: data/ui/sbackup-config.glade:613
msgid "<b>Configuration profiles</b>"
msgstr "<b>Profili di configurazione</b>"

#: data/ui/sbackup-config.glade:785
msgid ""
"<b>Add new profile</b>\n"
"\n"
"Please enter the new profile name and set the according checkbox in order to "
"enable resp. disable it."
msgstr ""

#: data/ui/sbackup-config.glade:810
msgid "Profile name"
msgstr ""

#: data/ui/sbackup-config.glade:836
msgid "Enabled"
msgstr ""

#: data/ui/sbackup-config.glade:915 data/ui/sbackup-restore.glade:7
msgid "Simple Backup Suite"
msgstr ""

#: data/ui/sbackup-config.glade:932
msgid "_File"
msgstr "_File"

#: data/ui/sbackup-config.glade:971
msgid "Save as..."
msgstr ""

#: data/ui/sbackup-config.glade:1008
msgid "_Tools"
msgstr "S_trumenti"

#: data/ui/sbackup-config.glade:1016 data/ui/sbackup-config.glade:1140
msgid "Set default settings..."
msgstr ""

#: data/ui/sbackup-config.glade:1037
msgid "Profile Manager..."
msgstr ""

#: data/ui/sbackup-config.glade:1058
msgid "Make backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1079
msgid "_Help"
msgstr "A_iuto"

#: data/ui/sbackup-config.glade:1087
msgid "_View all help topics"
msgstr ""

#: data/ui/sbackup-config.glade:1103
msgid "_About this program"
msgstr ""

#: data/ui/sbackup-config.glade:1154
msgid "Reload configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1168
msgid "Save configuration"
msgstr ""

#: data/ui/sbackup-config.glade:1191
msgid "Make a backup now"
msgstr ""

#: data/ui/sbackup-config.glade:1214
msgid "Open the profile manager"
msgstr ""

#: data/ui/sbackup-config.glade:1215
msgid "Profiles"
msgstr "Profili"

#: data/ui/sbackup-config.glade:1253
msgid ""
"none\n"
"gzip\n"
"bzip2"
msgstr ""
"Nessuno\n"
"gzip\n"
"bzip2"

#: data/ui/sbackup-config.glade:1271
msgid ""
"Compression format\n"
"Default is set to uncompressed backups."
msgstr ""

#: data/ui/sbackup-config.glade:1290 data/ui/sbackup-config.glade:1403
msgid ""
"You can split the snapshot to make backup on some filesystem that doesn't "
"support large files. This functionality can not be used if the backup is "
"compressed."
msgstr ""

#: data/ui/sbackup-config.glade:1330
msgid "Do a full backup at least once every"
msgstr "Eseguire un backup integrale almeno ogni:"

#: data/ui/sbackup-config.glade:1339
msgid ""
"Unless a set number of days has passed since the last full backup, "
"incremental backups will be done."
msgstr ""

#: data/ui/sbackup-config.glade:1356 data/ui/sbackup-config.glade:3015
msgid "days"
msgstr "giorni"

#: data/ui/sbackup-config.glade:1370
msgid " "
msgstr " "

#: data/ui/sbackup-config.glade:1380 data/ui/sbackup-config.glade:2323
msgid "MiB"
msgstr "MiB"

#: data/ui/sbackup-config.glade:1405
msgid "Split backup archives into several chunks"
msgstr ""

#: data/ui/sbackup-config.glade:1438 data/ui/sbackup-config.glade:2627
#: data/ui/sbackup-config.glade:2863 data/ui/sbackup-config.glade:3111
#: data/ui/sbackup-config.glade:3856
msgid "Help"
msgstr ""

#: data/ui/sbackup-config.glade:1442 data/ui/sbackup-config.glade:2632
msgid "Show help topic about configuration GUI."
msgstr ""

#: data/ui/sbackup-config.glade:1475
msgid "General"
msgstr "Generale"

#: data/ui/sbackup-config.glade:1495
msgid "<b>Included files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1570 data/ui/sbackup-config.glade:1818
#: data/ui/sbackup-config.glade:2023 data/ui/sbackup-config.glade:2181
msgid "Remove"
msgstr ""

#: data/ui/sbackup-config.glade:1620 data/ui/sbackup-config.glade:1868
msgid "Add File"
msgstr "Aggiungi file"

#: data/ui/sbackup-config.glade:1671 data/ui/sbackup-config.glade:1919
msgid "Add Directory"
msgstr "Aggiungi directory"

#: data/ui/sbackup-config.glade:1718
msgid "Include"
msgstr "Includi"

#: data/ui/sbackup-config.glade:1739
msgid "<b>Excluded files and directories</b>"
msgstr ""

#: data/ui/sbackup-config.glade:1952
msgid "Paths"
msgstr "Percorsi"

#: data/ui/sbackup-config.glade:2074 data/ui/sbackup-config.glade:2232
msgid "Add"
msgstr ""

#: data/ui/sbackup-config.glade:2109
msgid "File types"
msgstr "Tipi di file"

#: data/ui/sbackup-config.glade:2267
msgid ""
"Regular\n"
"Expressions"
msgstr ""

#: data/ui/sbackup-config.glade:2288
msgid "Do not backup files bigger than"
msgstr ""

#: data/ui/sbackup-config.glade:2344
msgid "Follow symbolic links"
msgstr ""

#: data/ui/sbackup-config.glade:2373
msgid "Others"
msgstr "Altri"

#: data/ui/sbackup-config.glade:2400
msgid "Exclude"
msgstr "Escludi"

#: data/ui/sbackup-config.glade:2421
msgid "Use default backup directory"
msgstr ""

#: data/ui/sbackup-config.glade:2446
msgid "/var/backup"
msgstr ""

#: data/ui/sbackup-config.glade:2472
msgid "Use custom local backup directory"
msgstr "Usare la directory di backup personalizzata"

#: data/ui/sbackup-config.glade:2500
msgid "Select the destination folder"
msgstr "Selezionare la cartella di destinazione"

#: data/ui/sbackup-config.glade:2526
#, fuzzy
msgid "Use a remote site"
msgstr "Aggiungi una risorsa remota"

#: data/ui/sbackup-config.glade:2568
msgid "ssh://username:password@example.com/remote/dir/"
msgstr "ssh://nomeutente:password@esempio.it/directory/remota/"

#: data/ui/sbackup-config.glade:2580
msgid "Connect..."
msgstr ""

#: data/ui/sbackup-config.glade:2672
msgid ""
"<i>The selected destination is not writeable with current permissions. "
"Backups cannot be stored there.</i>"
msgstr ""

#: data/ui/sbackup-config.glade:2704
msgid "Destination"
msgstr "Destinazione"

#: data/ui/sbackup-config.glade:2725
msgid "_No scheduled backups"
msgstr ""

#: data/ui/sbackup-config.glade:2742
msgid "_Simple"
msgstr ""

#: data/ui/sbackup-config.glade:2747
#, fuzzy
msgid ""
"This option will make a backup not at a precise time but when the computer "
"is up. "
msgstr ""
"Questa opzione fa si che sbackup faccia il backup quando il computer è "
"acceso e non ad un orario preciso. "

#: data/ui/sbackup-config.glade:2761
msgid "_Custom"
msgstr ""

#: data/ui/sbackup-config.glade:2781
msgid "Do backups"
msgstr ""

#: data/ui/sbackup-config.glade:2798
msgid "Cron time definition"
msgstr ""

#: data/ui/sbackup-config.glade:2834
msgid ""
"hourly\n"
"daily\n"
"weekly\n"
"monthly"
msgstr ""

#: data/ui/sbackup-config.glade:2867
msgid "Show help topic about scheduled backups."
msgstr ""

#: data/ui/sbackup-config.glade:2905
msgid ""
"<small>You can configure scheduled (i.e. automated) backups either simply by "
"choosing an interval or precisely by a custom cron definition. See \"man 5 "
"crontab\" for cron time definition format.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:2937
msgid "Schedule"
msgstr ""

#: data/ui/sbackup-config.glade:2955
#, fuzzy
msgid "Enable purging of old backups"
msgstr "Abilitare l'eliminazione dei backup vecchi e incompleti"

#: data/ui/sbackup-config.glade:2983
msgid "Simple cut-off: Erase all backups older than"
msgstr ""

#: data/ui/sbackup-config.glade:3039
msgid "Logarithmic"
msgstr "Logaritmica"

#: data/ui/sbackup-config.glade:3065
msgid ""
"<small>Keep progressively less backups into the past:\n"
"Keep all backups from yesterday.\n"
"Keep one backup per day from last week.\n"
"Keep one backup per week from last month.\n"
"Keep one backup per month from last year.\n"
"Keep one backup per year further into past.\n"
"Erase all other backups.</small>"
msgstr ""

#: data/ui/sbackup-config.glade:3116
msgid "Show help topic about purging of backups."
msgstr ""

#: data/ui/sbackup-config.glade:3152
msgid ""
"<small>Note: Additionally all incomplete and irrecoverable backups will be "
"erased as long as there is a complete backup snapshot made later on.</small>"
msgstr ""
"<small>Nota: gli ulteriori backup incompleti e irrecuperabili verranno "
"cancellati finché vi sarà sempre un backup completo.</small>"

#: data/ui/sbackup-config.glade:3184
msgid "Purging"
msgstr "Pulizia"

#: data/ui/sbackup-config.glade:3186
msgid "Cleaning old backups"
msgstr "Eliminazione vecchi backup"

#: data/ui/sbackup-config.glade:3209
msgid "<b>Logging</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3234
msgid "Debug is just for debbuging purpose.Info is the recommended setting"
msgstr "Debug è solo per il debug. L'impostazione raccomandata è Info"

#: data/ui/sbackup-config.glade:3236
msgid "Log level"
msgstr ""

#: data/ui/sbackup-config.glade:3248
msgid "Debug is just for debbuging purpose. Info is the recommended setting."
msgstr ""

#: data/ui/sbackup-config.glade:3274
msgid "Log file directory"
msgstr "Directory del file del registro"

#: data/ui/sbackup-config.glade:3320
msgid "<b>Report by mail</b>"
msgstr ""

#: data/ui/sbackup-config.glade:3350
msgid "From"
msgstr ""

#: data/ui/sbackup-config.glade:3361
msgid "SMTP server"
msgstr ""

#: data/ui/sbackup-config.glade:3375 data/ui/sbackup-config.glade:4053
msgid "Port"
msgstr ""

#: data/ui/sbackup-config.glade:3391
msgid "To"
msgstr ""

#: data/ui/sbackup-config.glade:3478
msgid "Use a username and password"
msgstr "Utilizzare nome utente e password"

#: data/ui/sbackup-config.glade:3508 data/ui/sbackup-config.glade:4075
msgid "Username"
msgstr "Nome utente"

#: data/ui/sbackup-config.glade:3540 data/ui/sbackup-config.glade:4124
msgid "Password"
msgstr "Password"

#: data/ui/sbackup-config.glade:3584
msgid "Use secure connection"
msgstr ""

#: data/ui/sbackup-config.glade:3615
msgid "Choose a certificate file"
msgstr ""

#: data/ui/sbackup-config.glade:3629
msgid "Key"
msgstr "Chiave"

#: data/ui/sbackup-config.glade:3645
msgid "Certificate"
msgstr "Certificato"

#: data/ui/sbackup-config.glade:3659
msgid "Choose a key file"
msgstr ""

#: data/ui/sbackup-config.glade:3731
msgid "Test _mail settings"
msgstr ""

#: data/ui/sbackup-config.glade:3772
msgid "Report"
msgstr "Rapporto"

#: data/ui/sbackup-config.glade:3811
msgid "Warning"
msgstr ""

#: data/ui/sbackup-config.glade:3860
msgid "Show help topic about default values."
msgstr ""

#: data/ui/sbackup-config.glade:3893
msgid "_Set"
msgstr ""

#: data/ui/sbackup-config.glade:3935
#, fuzzy
msgid "Set remote target"
msgstr "Inserire la destinazione remota da includere"

#: data/ui/sbackup-config.glade:3957
msgid "Type of service"
msgstr ""

#: data/ui/sbackup-config.glade:3997
#, fuzzy
msgid "Hostname"
msgstr "Nome utente"

#: data/ui/sbackup-config.glade:4027
#, fuzzy
msgid "<b>Details</b>"
msgstr "<b>Dettagli dell'immagine</b>"

#: data/ui/sbackup-config.glade:4157
#, fuzzy
msgid "Show password"
msgstr "Password"

#: data/ui/sbackup-config.glade:4217
#, fuzzy
msgid "_Connect"
msgstr "Contenuto"

#: data/ui/sbackup-config.glade:4293
msgid "Make a full backup"
msgstr ""

#: data/ui/sbackup-restore.glade:38
#, fuzzy
msgid "<b>Backup destination</b>"
msgstr "Ripristino di sbackup"

#: data/ui/sbackup-restore.glade:52
msgid "Displays the manual in Help browser."
msgstr ""

#: data/ui/sbackup-restore.glade:85
#, fuzzy
msgid "Default profile's destination"
msgstr "Profilo predefinito"

#: data/ui/sbackup-restore.glade:90
msgid ""
"Set backup destination to the place specified in default configuration "
"profile"
msgstr ""

#: data/ui/sbackup-restore.glade:137
#, fuzzy
msgid "Custom destination"
msgstr "Destinazione"

#: data/ui/sbackup-restore.glade:143
msgid ""
"Select a backup destination on your harddisk or at a remote site. Press "
"Apply when ready."
msgstr ""

#: data/ui/sbackup-restore.glade:184
msgid "Choose a destination folder. For remote sites fill URI in text field"
msgstr ""

#: data/ui/sbackup-restore.glade:201
msgid "Change destination to path specified in left text field"
msgstr ""

#: data/ui/sbackup-restore.glade:266
msgid "Select a bold printed date to get the list of backups for that date."
msgstr ""

#: data/ui/sbackup-restore.glade:310
#, fuzzy
msgid "Available backups"
msgstr "<b>Backup disponibili</b>"

#: data/ui/sbackup-restore.glade:398
#, fuzzy
msgid "Delete snapshot"
msgstr "Eliminare l'immagine corrente?"

#: data/ui/sbackup-restore.glade:436
msgid "Delete"
msgstr "Elimina"

#: data/ui/sbackup-restore.glade:475
#, fuzzy
msgid "Upgrade snapshot"
msgstr "Impossibile aggiornare l'immagine"

#: data/ui/sbackup-restore.glade:515
msgid "Upgrade"
msgstr "Aggiorna"

#: data/ui/sbackup-restore.glade:553
msgid "<b>Snapshot History</b>"
msgstr "<b>Storia dell'immagine</b>"

#: data/ui/sbackup-restore.glade:579
msgid "Current base:"
msgstr ""

#: data/ui/sbackup-restore.glade:620
msgid "Select a snapshot on this list to rebase on it."
msgstr "Selezionare un'immagine nell'elenco per creare la nuova base."

#: data/ui/sbackup-restore.glade:695
msgid "Rebase"
msgstr "Creare la nuova base"

#: data/ui/sbackup-restore.glade:746
msgid "<b>Snapshot Management</b>"
msgstr "<b>Gestione immagine</b>"

#: data/ui/sbackup-restore.glade:785
#, fuzzy
msgid "Snapshot details"
msgstr "<b>Dettagli dell'immagine</b>"

#: data/ui/sbackup-restore.glade:850 data/ui/sbackup-restore.glade:867
msgid "Restores files/directories from the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:862
msgid "Restore"
msgstr "Ripristina"

#: data/ui/sbackup-restore.glade:882 data/ui/sbackup-restore.glade:901
msgid ""
"Restores files/directories recursively starting from the base snapshot to "
"the selected snapshot"
msgstr ""

#: data/ui/sbackup-restore.glade:896
msgid "Revert"
msgstr "Reimposta"

#: data/ui/sbackup-restore.glade:939
msgid "<b>Restoration Management</b>"
msgstr "<b>Gestione ripristino</b>"

#: data/ui/sbackup-restore.glade:964
msgid "<b>Export Management</b>"
msgstr "<b>Gestione esportazione</b>"

#: data/ui/sbackup-restore.glade:1056
msgid "Headline"
msgstr "Intestazione"

#: data/ui/sbackup-restore.glade:1069
msgid "Content"
msgstr "Contenuto"

#: scripts/sbackup-upgrade-backups.py:45
msgid ""
"\n"
"Simple Backup suit command line backup format upgrade\n"
"Usage: upgrade-backup backup-target-url\n"
"Note: backup-target-url must not include the snapshot subdirectory name, for "
"example:\n"
"\n"
"   /var/backup/\n"
"\n"
"Use simple-restore-gnome for more ease of use.\n"
msgstr ""
"\n"
"Simple Backup Suite strumento a linea di comando per l'aggiornamento del "
"backup\n"
"Uso: upgrade-backup backup-target-url\n"
"Nota: backup-target-url non deve includere il nome della sotto directory "
"dell'immagine, per esempio:\n"
"   /var/backup/\n"
"\n"
"Usare simple-restore-gnome per una semplicità ancora maggiore.\n"

#~ msgid "Failed to set the gid to 'admin' one :"
#~ msgstr "Impostazione del gid amministrativo non riuscita:"

#~ msgid "Target not writeable: "
#~ msgstr "Oggetto non riscrivibile: "

#~ msgid "Invalid value of the target directory : "
#~ msgstr "Valore non valido della directory di destinazione: "

#~ msgid ""
#~ "Got a non valid snapshot '%(name)s' due to name convention : %"
#~ "(error_cause)s "
#~ msgstr ""
#~ "Ottenuta un'immagine non valida «%(name)s» a causa della convenzione sul "
#~ "nome: %(error_cause)s "

#~ msgid "Cannot downgrade other format than 'gzip' to 1.4"
#~ msgstr "Impossibile riporatre alla versione 1.4 formati che non siano «gzip»"

#~ msgid "'%d' test(s) didn't succeed !"
#~ msgstr "La prova «%d» non ha avuto successo."

#~ msgid ""
#~ "Please test writability of the target directory by pressing \"Test\" "
#~ "button on the right."
#~ msgstr ""
#~ "Controllare che la directory di destinazione sia accessibile premendo il "
#~ "pulsante «Prova»."

#~ msgid "Test Succeeded !"
#~ msgstr "Prova superata con successo."

#~ msgid "Target directory is writable."
#~ msgstr "È possibile scrivere nella directory di destinazione."

#~ msgid ""
#~ "Please change target directory and test writability of the target "
#~ "directory by pressing \"Test\" button on the right."
#~ msgstr ""
#~ "Cambiare la directory di destinazione e controllare di avere i permessi "
#~ "di scrittura facendo clic sul pulsante «Prova» sulla destra."

#~ msgid "disable"
#~ msgstr "disabilita"

#~ msgid ""
#~ "You can't %s the Default Profile. Please use it if you need only one "
#~ "profile."
#~ msgstr ""
#~ "Non puoi %s il Profilo di Default. Per favore, usa, se ne hai bisogno, "
#~ "solo un profilo."

#~ msgid "Actual base : <b>%s</b>"
#~ msgstr "Base attuale: <b>%s</b>"

#~ msgid ""
#~ "Choose the compression format that you want to apply.\n"
#~ "Default is <b>gzip</b>."
#~ msgstr ""
#~ "Scegliere il formato di compressione da appllicare.\n"
#~ "Quello predefinito è <b>gzip</b>."

#~ msgid "Use a remote directory (SSH or FTP)"
#~ msgstr "Usare una directory remota (SSH o FTP)"

#~ msgid "Test"
#~ msgstr "Prova"

#~ msgid ""
#~ "Select the directory in which you have your backups.\n"
#~ "You can use a <b>local</b> or <b>distant</b> directory depending on the "
#~ "plugins you have installed."
#~ msgstr ""
#~ "Selezionare la directory in cui si trovano i backup.\n"
#~ "È possibile utilizzare una directory <b>locale</b> o <b>remota</b> a "
#~ "seconda dei plugin installati."

#~ msgid "Use the default directory : "
#~ msgstr "Utilizzare la directory predefinita : "

#~ msgid "Use a custom directory : "
#~ msgstr "Utilizzare una directory personalizzata : "

#~ msgid "<b>Snapshots directory</b>"
#~ msgstr "<b>Directory delle immagini</b>"

#~ msgid "Restore As"
#~ msgstr "Ripristina come"

#~ msgid "Revert As"
#~ msgstr "Reimposta come"

#~ msgid "Choose to upgrade your snapshot if needed."
#~ msgstr "Scegliere di aggiornare l'immagine se necessario."

#~ msgid "Actual base :"
#~ msgstr "Base attuale:"

#~ msgid "Select a snapshot and change its base to another snapshot."
#~ msgstr "Selezionare un'immagine e cambiarne la base a un'altra."

#~ msgid "Use default backup directory (/var/backup)"
#~ msgstr "Usare la directory predefinita (/var/backup)"

#~ msgid "Use recommended backup settings"
#~ msgstr "Usare le impostazioni di backup raccomandate"

#~ msgid "Use custom backup settings"
#~ msgstr "Usare le impostazioni personalizzate"

#~ msgid ""
#~ "<small>Do backups according to the settings defined on the next pages of "
#~ "this dialog.</small>"
#~ msgstr ""
#~ "<small>Esegue backup con le impostazioni definite dall'utente in questa "
#~ "finestra.</small>"

#~ msgid ""
#~ "<small>Do not backup your data automatically. Note: this does not prevent "
#~ "doing manual backups.</small>"
#~ msgstr ""
#~ "<small>Non esegue backup automatici. Nota: ciò non impedisce l'esecuzione "
#~ "di backup manuali.</small>"

#~ msgid "Mb"
#~ msgstr "Mb"

#~ msgid "at:"
#~ msgstr "a:"

#~ msgid "Day of month:"
#~ msgstr "Giorno del mese:"

#~ msgid "Day of week:"
#~ msgstr "Giorno della settimana:"

#~ msgid "Hour"
#~ msgstr "Ora"

#~ msgid "Minute"
#~ msgstr "Minuto"

#~ msgid "Parameters"
#~ msgstr "Parametri"

#~ msgid "This is a user friendly backup solution for common desktop needs."
#~ msgstr "È un'applicazione di facile utilizzo per creare copie di backup."

#~ msgid "sbackup configuration error"
#~ msgstr "Errore di configurazione di sbackup"

#~ msgid "Unable to change permissions for "
#~ msgstr "Impossibile cambiare i permessi per "

#~ msgid "I didn't find the log file. Please set it up in sbackup.conf "
#~ msgstr ""
#~ "Impossibile trovare il file del registro. Impostarlo in sbackup.conf "

#~ msgid "Choose the SSL option (smtptls=1) to be able to use Cert and Key"
#~ msgstr ""
#~ "Scegliere l'opzione SSL (smtptls=1) per poter utilizzare certificato e "
#~ "chiave"

#~ msgid "Save to file ..."
#~ msgstr "Salvare  nel file..."

#~ msgid "Starting snapshot %(name)s "
#~ msgstr "Avvio dell'immagine %(name)s "

#~ msgid "gtk-no"
#~ msgstr "gtk-no"

#~ msgid "gtk-yes"
#~ msgstr "gtk-yes"

#~ msgid "Test all"
#~ msgstr "Prova tutto"

#~ msgid ""
#~ "When setting a username (resp password), password (resp username) is "
#~ "mandatory"
#~ msgstr ""
#~ "Quando si imposta nome utente o password è obbligatorio inserire entrambi"

#~ msgid "No directory to backup !"
#~ msgstr "Nessuna directory per il backup."

#~ msgid "Target '%s' directory is excluded"
#~ msgstr "La directory di destinazione «%s» è esclusa"

#~ msgid "'%(file)s' doesn't exist, it has to be exclude "
#~ msgstr "«%(file)s» non esiste, è da escludere "

#~ msgid "Initializing FUSE FILE ACCESS MANAGER !"
#~ msgstr "Avvio gestore di accesso al file fuse."

#~ msgid "Got an exception when Pulling '%s' : "
#~ msgstr "Ottenuta una eccezione prelevando «%s»: "

#~ msgid "Deleting '%(snp)s' for purge !"
#~ msgstr "Eliminazione «%(snp)s» per pulizia."

#~ msgid "Couldn't make a proper backup : "
#~ msgstr "Impossibile fare un backup corretto: "

#~ msgid "Snapshot '%s' is a full . Can't rebase on older snapshot"
#~ msgstr ""
#~ "L'immagine «%s» è piena. Impossibile creare la nuova base su un'immagine "
#~ "precedente"

#~ msgid "Terminating FUSE FILE ACCESS MANAGER !"
#~ msgstr "Chiusura gestore di accesso al file fuse."

#~ msgid "Problem with '%(file)s' : %(error)s "
#~ msgstr "Problema con «%(file)s» : %(error)s "

#~ msgid "Non existing directory : %s"
#~ msgstr "Directory inesistente: %s"

#~ msgid "gtk-remove"
#~ msgstr "gtk-remove"

#~ msgid "gtk-add"
#~ msgstr "gtk-add"

#~ msgid "gtk-close"
#~ msgstr "gtk-close"

#~ msgid "gtk-edit"
#~ msgstr "gtk-edit"

#~ msgid ""
#~ "Please enter the new profile name in the <b>Profile name</b> entry and "
#~ "check the <b>Enable</b> checkbox to enable it."
#~ msgstr ""
#~ "Inserire il nuovo nome del profilo nella casella <b>Nome profilo</b> e "
#~ "selezionare <b>Abilita</b> per attivarlo."

#~ msgid "gtk-ok"
#~ msgstr "gtk-ok"

#~ msgid "gtk-cancel"
#~ msgstr "gtk-cancel"

#~ msgid "Regex"
#~ msgstr "Espressione regolare"

#~ msgid "precisely"
#~ msgstr "preciso"

#~ msgid "simply"
#~ msgstr "semplice"

#~ msgid "Time (for admins only)"
#~ msgstr "Ora (solo per gli amministratori)"

#~ msgid "Log settings"
#~ msgstr "Impostazioni del registro"

#~ msgid "Security"
#~ msgstr "Sicurezza"

#~ msgid ""
#~ "We don't have read access to '%(file)s', it has to be exclude : %(error)s "
#~ msgstr ""
#~ "Impossibile accedere in lettura a «%(file)s», è da escludere : %(error)s "

#~ msgid ""
#~ "'%(file)s' size is higher than the specified one ( %(filesize)s > %"
#~ "(maxsize)s), it has to be exclude "
#~ msgstr ""
#~ "«%(file)s» ha una dimensione maggiore di quella specificata (%(filesize)s "
#~ "> %(maxsize)s), è da escludere "

#~ msgid "Set the reciever of this mail"
#~ msgstr "Impostare il destinatario di questa mail"

#~ msgid ""
#~ "Debug is just for debbuging purpose\n"
#~ "Info is the recommended setting"
#~ msgstr ""
#~ "Debug è solo per il debug\n"
#~ "L'impostazione raccomandata è Info"

#~ msgid "Terminating FUSE FILE ACCESS MANAGER!"
#~ msgstr "In chiusura MANAGER ACCESSO FILE FUSO!"
